src/ui/components/Timeline/index.js

```diff
@@ -255,7 +255,8 @@ export class Timeline extends Component {
   };
 
   onPlayerMouseUp = e => {
-    const { hoverTime, hoveringOverMarker } = this.props;
+    const { hoverTime } = this.props;
+    const { hoveringOverMarker } = this.state;
```

Comment:
It would be nice to add this to the default state in the constructor()

Reply:
Good catch.